_search-plugins/sql/functions.md

```diff
@@ -18,166 +18,170 @@ The SQL plugin supports the following common functions shared across the SQL and
+| second             | `second(time) -> integer`                                                              | `SELECT second((time '01:02:03'))`                                                  |
+| second_of_minute\* | `second_of_minute(time) -> integer`                                                    | `SELECT second_of_minute((time '01:02:03'))`                                        |
+| sec_to_time\*      | `sec_to_time(integer) -> date`                                                         | `SELECT sec_to_time(10000)`                                                         |
+| subdate            | `subdate(date, INTERVAL expr unit) -> date, datetime`                                  | `SELECT subdate(date('2008-01-02'), INTERVAL 31 day)`                               |
+| subtime            | `subtime(date, date) -> date`                                                          | `SELECT subtime(date('2008-12-12'), date('2008-11-15'))`                            |
+| str_to_date\*      | `str_to_date(string, format) -> date`                                                  | `SELECT str_to_date("March 10 2000", %M %d %Y")`                                    |
+| time               | `time(expr) -> time`                                                                   | `SELECT time('13:49:00')`                                                           |
+| timediff           | `timediff(time, time) -> time`                                                         | `SELECT timediff(time('23:59:59'), time('13:00:00'))`                               |
+| timestamp          | `timestamp(date) -> date`                                                              | `SELECT timestamp('2001-05-07')`                                                    |
+| timestampadd       | `timestampadd(interval, integer, date) -> date)`                                       | `SELECT timestampadd(DAY, 17, datetime('2000-01-01 00:00:00'))`                     |
+| timestampdiff      | `timestampdiff(interval, date, date) -> integer`                                       | `SELECT timestampdiff(YEAR, '1997-01-01 00:00:00, '2001-03-06 00:00:00')`           |
+| time_format        | `time_format(date, string) -> string`                                                  | `SELECT time_format('1998-01-31 13:14:15.012345', '%f %H %h %I %i %p %r %S %s %T')` |
+| time_to_sec        | `time_to_sec(time) -> long`                                                            | `SELECT time_to_sec(time '22:23:00')`                                               |
+| to_days            | `to_days(date) -> long`                                                                | `SELECT to_days(date '2008-10-07')`                                                 |
+| to_seconds         | `to_seconds(date) -> integer`                                                          | `SELECT to_seconds(date('2008-10-07')`                                              |
```

Comment:
```suggestion
| to_seconds         | `to_seconds(date) -> integer`                                                          | `SELECT to_seconds(date('2008-10-07'))`                                              |
```

Reply:
fixed in https://github.com/opensearch-project/documentation-website/pull/4232/commits/9442d3b1e8ec65a7c0e9b166073cf2b8281553d1